testsuite/grpc-tests/pom.xml

```diff
@@ -223,6 +229,15 @@
             </testResource>
         </testResources>
         <plugins>
+        <plugin>
+          <groupId>org.apache.maven.plugins</groupId>
+          <artifactId>maven-compiler-plugin</artifactId>
+          <configuration>
+			  <compilerArgs>
+                  <arg>-J--add-opens java.base/jdk.internal.misc=ALL-UNNAMED -J--add-opens java.base/java.util=ALL-UNNAMED</arg>
```

Comment:
What's this for?

Reply:
I think it fixed a problem at some point. It doesn't seem to be needed now. Gone.